src/cli.ts

```diff
@@ -7,8 +7,8 @@ const P = new Pokedex();
 const fetchPokemonInfo = async (name: string) => {
   console.log(chalk.green(`Fetching information about ${name}...`));
   try {
-    const pokemonSpecies = await P.getPokemonByName(name.toLowerCase());
-    return pokemonSpecies;
+    const pokemonType = await P.getPokemonByName(name.toLowerCase()) as Pokedex.Pokemon;
```

Comment:
[nitpick] this is not actually a Pokemon type yet, this is still just a `Pokemon`. So I would name this constant `pokemon`, not `pokemonType`. (I'm talking about Pokemon types like "fire" and "grass", not a TypeScript type!)